LICENSE

```diff
@@ -0,0 +1,21 @@
+Licensed under the standard MIT license:
+
+Copyright 2015 Nate Smith
```

Comment:
```suggestion
Copyright 2021 Nate Smith
```

Reply:
> The general rule is that the year to include in a copyright notice is the year of first publication of the work. 

https://www.copyrightlaws.com/copyright-symbol-notice-year/#h-which-year-should-you-use-in-your-copyright-notice

Reply:
Oh... TIL